### PR TITLE
Update automation pipeline check retries

### PR DESCRIPTION
**Pull request by william-msft**
- [x] The purpose of this PR is explained in this message or in an issue. If an issue please include a reference as:
  - I've noticed our pipelines are increasingly taking longer to run, which has started to mess with Automation. I've increased the number of retries to mitigate this obstacle. [Example](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=8371936&view=results):
    -  image
~- [ ] Tests are included and/or updated for code changes.~
~- [ ] Proper license headers are included in each file.~

### CODE REVIEW

**Review by cormacpayne — APPROVED** [review]
:shipit:

File: .github/workflows/automationTemplate.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ on:
         description: 'The platform name should be all lowercase'
 
 env:
-  MAX_RETRIES: 10
+  MAX_RETRIES: 15
```
Comment (waliMSFT) [inline .github/workflows/automationTemplate.yaml:19]: I am wondering how increasing the number of retries will mitigate the issue?

Comment (william-msft) [reply .github/workflows/automationTemplate.yaml:19]: If the he pipeline does not finish within 1.5hrs then the pipeline will fail. Currently its taking more than that (shown in the picture of the description) takes over 2 hours.

Comment (william-msft) [reply .github/workflows/automationTemplate.yaml:19]: If the pipeline does not finish within 1.5hrs then the pipeline will fail. Currently its taking more than that (shown in the picture of the description) takes over 2 hours.

**Pull request merged**